lib/beeline/health_checker.ex

```diff
@@ -123,7 +126,8 @@ defmodule Beeline.HealthChecker do
       interval: state.interval,
       drift: state.drift,
       measurement_time: DateTime.utc_now(),
-      prior_position: state.current_position
+      prior_position: state.current_position,
+      auto_subscribe: state.auto_subscribe?
```

Comment:
wasn't sure if we want this to be `auto_subscribe` or `auto_subscribe?` just following `health_checker` pattern